.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @microsoft/vscodearmtoolsdev @StephenWeatherford @alexgav @bhsubra
+* @microsoft/vscodearmtoolsdev @StephenWeatherford @alexgav
```

Comment:
>alexgav [](start = 52, length = 7)

She's going to be back in May :)

Reply:
Okay, if she's coming back to the team, I'll leave her it for now.